test/powershell/Modules/Microsoft.PowerShell.Utility/MarkdownCmdlets.Tests.ps1

```diff
@@ -0,0 +1,335 @@
+            }
+
+            $output.VT100EncodedString | Should -BeExactly $expectedString
+        }
+
+        It 'Can convert element : <element> to HTML using pipeline input' -TestCases $TestCases {
+            param($element, $inputMD, $text, $codeFormatString, $codeText, $url)
+
+            $output = $inputMD | ConvertFrom-Markdown
+
+            if($element -like 'Header?' -or
+               $element -eq 'Bold' -or
+               $element -eq 'Italics')
+            {
+                $expectedString = GetExpectedHTML -ElementType $element -Text $text
+            }
+            elseif($element -eq 'Code')
+            {
+                $expectedString = GetExpectedHTML -ElementType $element -CodeFormatString $codeFormatString -CodeText $codeText
+            }
+            elseif($element -eq 'CodeBlock')
+            {
+                $expectedString = GetExpectedHTML -ElementType $element -CodeText $codeText
+            }
+            elseif ($element -eq 'Link')
+            {
+                $expectedString = GetExpectedHTML -ElementType $element -Text $text -Url $url
+            }
+            elseif ($element -eq 'Image')
+            {
+                $expectedString = GetExpectedHTML -ElementType $element -Text $text -Url $url
+            }
+
+            $output.Html | Should -BeExactly $expectedString
+        }
+
+        It 'Can convert input from a file path to vt100 encoded string' {
+            $output = ConvertFrom-Markdown -Path $mdFile.FullName -AsVT100EncodedString
+            $output.VT100EncodedString | Should -BeExactly $expectedStringFromFile
+        }
+
+        It 'Can convert input from a fileinfo object to vt100 encoded string' {
+            $ouputFromFileInfo = $mdFile | ConvertFrom-Markdown -AsVT100EncodedString
+            $ouputFromFileInfo.VT100EncodedString | Should -BeExactly $expectedStringFromFile
+        }
+
+        It 'Can convert input from a literal path to vt100 encoded string' {
+            $output = ConvertFrom-Markdown -Path $mdLiteralPath -AsVT100EncodedString
+            $output.VT100EncodedString | Should -BeExactly $expectedStringFromFile
+        }
+    }
+
+    Context "ConvertFrom-Markdown error cases" {
+        It "Gets an error if path is not FileSystem provider path" {
+            { ConvertFrom-Markdown -Path Env:\PSModulePath -ErrorAction Stop } | Should -Throw -ErrorId 'OnlyFileSystemPathsSupported,Microsoft.PowerShell.Commands.ConvertFromMarkdownCommand'
+        }
+
+        It "Gets an error if path does not exist" {
+            { ConvertFrom-Markdown -Path DoestnotExist -ErrorAction Stop } | Should -Throw -ErrorId 'FileNotFound,Microsoft.PowerShell.Commands.ConvertFromMarkdownCommand'
+        }
+
+        It "Gets an error if input object type is not correct" {
+            { ConvertFrom-Markdown -InputObject 1 -ErrorAction Stop } | Should -Throw -ErrorId 'InvalidInputObject,Microsoft.PowerShell.Commands.ConvertFromMarkdownCommand'
+        }
+    }
```

Comment:
the scenario I mentioned above was this scenario:
`[io.fileinfo]::new("lol-I-dont-exist") | ConvertFrom-Markdown`
it's a pretty corner-case scenario, I'll admit

Reply:
Added test.